### PR TITLE
Update container image workflow for Windows Server 2019 and 1809.

### PR DESCRIPTION
**Pull request by pjh**
Microsoft made several changes to the naming and location of their base
Windows containers with the release of Windows Server 2019 and 1809.
This change adds workflow files for Windows Server 2019 and 1809 and
updates the container image installation script to accommodate the
version differences.

====Tested:====
time ~/go/bin/daisy -project PROJECT -zone us-west1-b daisy_workflows/image_build/windows_for_containers/windows-1709-for-containers.wf.json
time ~/go/bin/daisy -project PROJECT -zone us-west1-b daisy_workflows/image_build/windows_for_containers/windows-1803-for-containers.wf.json
time ~/go/bin/daisy -project PROJECT -zone us-west1-b daisy_workflows/image_build/windows_for_containers/windows-1809-for-containers.wf.json
time ~/go/bin/daisy -project PROJECT -zone us-west1-b daisy_workflows/image_build/windows_for_containers/windows-2019-for-containers.wf.json
time ~/go/bin/daisy -project PROJECT -zone us-west1-b daisy_workflows/image_build/windows_for_containers/windows-2019-core-for-containers.wf.json

### CODE REVIEW

**Comment by pjh** [comment]
@adjackura please take a look!

CC @rofuentes

File: daisy_workflows/image_build/windows_for_containers/container_image_install.ps1 [hunk 1]
```diff
@@ -47,61 +47,163 @@ function Get-MetadataValue {
...
+  Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+
+  Write-Host 'Installing DockerMsftProvider module'
+  Install-Module -Name DockerMsftProvider -Repository PSGallery -Force
+
+  $docker_version = "18.09"
+  Write-Host "Installing Docker EE ${docker_version}"
+  Install-Package -Name docker -ProviderName DockerMsftProvider -Force -RequiredVersion ${docker_version}
+
+  Write-Host 'Enabling IPv6'
```
Comment (adjackura) [inline daisy_workflows/image_build/windows_for_containers/container_image_install.ps1]: I do not believe we need this anymore as the base images don't disable ipv6

File: daisy_workflows/image_build/windows_for_containers/container_image_install.ps1 [hunk 2]
```diff
@@ -47,61 +47,163 @@ function Get-MetadataValue {
...
+  $label = Get-ContainerVersionLabel $windows_version
+  return "${repo}/${image}:${label}"
+}
+
+function Get-BaseContainerImageNames {
+  param (
+    [parameter(Mandatory=$true)]
+      [string]$windows_version
+  )
+  $images = @(Get-ServerCoreImageName $windows_version)
```
Comment (adjackura) [inline daisy_workflows/image_build/windows_for_containers/container_image_install.ps1]: What about mcr.microsoft.com/windows:1809, is this a new container?

File: daisy_workflows/image_build/windows_for_containers/container_image_install.ps1 (hunk 1 above)
Comment (pjh) [reply daisy_workflows/image_build/windows_for_containers/container_image_install.ps1]: Ok, added a commit that removes this.

File: daisy_workflows/image_build/windows_for_containers/container_image_install.ps1 (hunk 2 above)
Comment (pjh) [reply daisy_workflows/image_build/windows_for_containers/container_image_install.ps1]: I believe that's the full-blown Windows container that supports every Windows API. Microsoft announced this at Build earlier this year and is making it available with 1809 and 2019. I don't know exactly how big it is but it's probably huge. I don't think we need to add it to our for-containers images at this time but let me know if you think we should.

**Pull request merged**